rubrics/api/src/main/java/org/sakaiproject/rubrics/logic/model/Metadata.java

```diff
@@ -27,6 +27,8 @@
 import javax.persistence.Embeddable;
 import javax.persistence.Transient;
 
+import org.sakaiproject.rubrics.logic.RubricsLocalDateTimeSerializer;
```

Comment:
Curious why dates are not sent using the ISO format and then converted to whatever is needed on the client.

Reply:
I don't know why, either. All I know is that they were being serialized in standard ISO format and that's how the ui code gets them. Alternatively, we could send them as epoch dates and let client side JS deal with it.